code/datums/flock/flock.dm

```diff
@@ -560,6 +570,21 @@ var/flock_signal_unleashed = FALSE
 	var/enemy_name = M
 	return (enemy_name in src.enemies)
 
+/datum/flock/proc/addAlly(atom/A)
```

Comment:
Think it would probably be safer to use same validity criteria that checking for enemies uses, like must be living

Reply:
Also could loop through occupants in vehicles to add them as allies

Reply:
that's already checked in the proc above, seems pointless to duplicate it here
I don't think you'd want to set everyone in a pod to ally, since you don't know who's in the pod and accidentally setting someone to ally is much more dangerous to you than accidentally setting them to enemy. Might be worth it for consistency though.